lib/vehicle_factory.rb

```diff
@@ -0,0 +1,8 @@
+class VehicleFactory
+  def create_vehicles(cars)
+     cars.map do |registration|
+       Vehicle.new(make: registration[:make], model: registration[:model], year: registration[:model_year],
```

Comment:
Nice job using `.map`. One this to remember is what kind of datatype does a new instantiation of `Vehicle` take? You do have it set up like a hash, but you are not passing it like a hash. 
```ruby
Vehicle.new({
   make: registration[:make], 
   model: registration[:model], 
   year: registration[:model_year],
   vin: registration[:vin_1_10], engine: :ev
})
```
You are missing the `{}` inside of your `()`.